kolibri/core/assets/src/vue/session-nav-widget/index.vue

```diff
@@ -16,13 +16,13 @@
     <div class="user-dropdown">
       <ul class="dropdown-list">
         <li>
-          <h4 class="dropdown-name">{{ name }}</h4>
+          <p class="dropdown-name">{{ name }}</p>
```

Comment:
Heading was not strictly necessary, and it was inserted creating improper heading nesting.